skore-remote-project/src/skore_remote_project/project/widget.py

```diff
@@ -0,0 +1,535 @@
+from typing import Any, Optional, Union, cast
+
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from IPython.display import clear_output, display
+from ipywidgets import widgets
+
+
+class ModelExplorerWidget:
+    """
+    Widget for interactive parallel coordinate plots of ML experiment metadata.
+
+    This class creates and manages interactive widgets and parallel coordinate plots
+    for visually exploring machine learning experiment results. It allows users to
+    filter and compare experiments across different metrics, learners, and datasets.
+
+    Parameters
+    ----------
+    dataframe : pd.DataFrame
+        The dataframe containing the experiment metadata with columns for metrics,
+        learners, datasets, and ML task types.
+    seed : int, default=0
+        Random seed for jittering categorical columns to improve visualization.
+
+    Attributes
+    ----------
+    current_fig : go.FigureWidget or None
+        The currently displayed plotly figure.
+    current_selection : dict
+        Dictionary containing the current user selection criteria.
+    """
+
+    _plot_width: int = 800
+    _metrics: dict[str, dict[str, Union[str, bool]]] = {
+        "fit_time": {
+            "name": "Fit Time",
+            "greater_is_better": False,
+            "type": "time",
+            "show": False,
+        },
+        "predict_time_test": {
```

Comment:
```suggestion
        "predict_time": {
```

To change, after https://github.com/probabl-ai/skore/pull/1632.